tools/build_defs/scala/scala.bzl

```diff
@@ -26,6 +26,40 @@ def _adjust_resources_path(path):
     return dir_1 + dir_2, rel_path
   return "", path
 
+def _build_nosrc_jar(ctx, buildijar):
+  res_cmd = ""
+  for f in ctx.files.resources:
+    c_dir, res_path = _adjust_resources_path(f.path)
+    change_dir = "-C " + c_dir if c_dir else ""
+    res_cmd = "\n{jar} uf {out} " + change_dir + " " + res_path
+  ijar_cmd = ""
+  if buildijar:
```

Comment:
Why have buildijar here if you are not using it?

Also the ijar should be just an empty jar here, no?